### ISSUE TITLE
feat(platform) External users need an easy way of giving us feedback when reviewing the catalogue for us (and later on, too)

### ISSUE THREAD

**Issue by EleanorHyde-UMCG** [molgenis/molgenis-emx2#3851]
Morris/Esther: we denken dat nuttiger is te zorgen dat laagdrempelig vragen  kunnen worden gesteld. Bijvoorbeeld een fastlane (chat scherm ofzo) naar zammad?

demo:
* [ ] button on the main menu with an inviting title e.g. "send us feedback"  or "contact us"
* [ ] then we link to contact page
* [ ] there there will be a form to fill in
* [ ] and also a link explaining molgenis-support@umcg.nl 
* [ ] explaining what we do with the contact

**Is your feature request related to a problem? Please describe.**
Soon we will be asking specific people outside GCC to review the new-look data catalogue and we need to give them an easy way of giving us feedback while they're working in the catalogue

**Describe the solution you'd like**
Ideally we will have a link which the user clicks to open up the local email system and send an email, and this link should be visible wherever you are in the SSR catalogue.

**Text of link**: "Click here to send us feedback"
**Action when link is clicked**: an email addressed to molgenis-support@umcg.nl is started in a separate pop-up. Subject: Feedback about the data catalogue. Everything else is left empty for the user to fill in themselves.

**Location of link**: 
![image](https://github.com/molgenis/molgenis-emx2/assets/80456579/1db5f65e-7176-495e-8f01-2b5aed94e014)

@mswertz @connoratrug @BrendaHijmans @timcadman  discuss!!!

**Comment by BrendaHijmans** [comment]
“Send us feedback” should be sufficient. “Click here” is implied. Will the feedback be sent to molgenis-support?Op 5 jun 2024 om 16:58 heeft EleanorHyde-UMCG ***@***.***> het volgende geschreven:﻿
Is your feature request related to a problem? Please describe.
Soon we will be asking specific people outside GCC to review the new-look data catalogue and we need to give them an easy way of giving us feedback while they're working in the catalogue
Describe the solution you'd like
Ideally we will have a link which the user clicks to open up the local email system and send an email, and this link should be visible wherever you are in the SSR catalogue.
Text of link: "Click here to send us feedback"
Action when link is clicked: an email addressed to ***@***.*** is started in a separate pop-up. Subject: Feedback about the data catalogue. Everything else is left empty for the user to fill in themselves.
Location of link:
image.png (view on web)
@mswertz @connoratrug @BrendaHijmans @timcadman  discuss!!!

**Comment by EleanorHyde-UMCG** [comment]
> “Send us feedback” should be sufficient. “Click here” is implied. Will the feedback be sent to molgenis-support?Op 5 jun 2024 om 16:58 heeft EleanorHyde-UMCG ***@***.***> het volgende geschreven:﻿ Is your feature request related to a problem? Please describe. Soon we will be asking specific people outside GCC to review the new-look data catalogue and we need to give them an easy way of giving us feedback while they're working in the catalogue Describe the solution you'd like Ideally we will have a link which the user clicks to open up the local email system and send an email, and this link should be visible wherever you are in the SSR catalogue. Text of link: "Click here to send us feedback" Action when link is clicked: an email addressed to ***@***.*** is started in a separate pop-up. Subject: Feedback about the data catalogue. Everything else is left empty for the user to fill in themselves. Location of link: image.png (view on web) @mswertz @connoratrug @BrendaHijmans @timcadman discuss!!! —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

Yes, Molgenis Support email address (is in the original text but I think github deletes email addresses?)

**Comment by mswertz** [comment]
goal is to enable any users to get into contact with us, either for support or to provide feedback. So indeed handover to support channel is all we need here. Ideally we would include link to what user currently sees.

the point @connoratrug makes is a seperate one. I believe we should do:
- A light weigh survey as suggested
- And specifically interviews with key users

**Comment by mswertz** [comment]
we propose a banner on top that has
"Welcome to new user interface. We hope you like it. Please let us know at molgenis-support@umcg.nl. Previous version here."

This will be temporary.

Meanwhile this story needs to be completed.

**Comment by EleanorHyde-UMCG** [comment]
Tweaked text:

Welcome to our new user interface. We hope you like it. If you have any technical comments, please let us know at molgenis-support@umcg.nl. The previous version is [here](https://data-catalogue-acc.molgeniscloud.org/catalogue/catalogue/#/).
